testsuite/tests/th/T14888.stderr

```diff
@@ -7,4 +7,4 @@ T14888.hs:18:23-59: Splicing expression
     "class T14888.Functor' (f_0 :: * -> *)
     where T14888.fmap' :: forall (a_1 :: *) (b_2 :: *) .
                           (a_1 -> b_2) -> f_0 a_1 -> f_0 b_2
-instance T14888.Functor' ((->) r_3 :: * -> *)"
+instance T14888.Functor' ((->) r_3)"
```

Comment:
This is kind of weird :slightly_smiling_face:

Reply:
AFAICT this change is a casualty of `->` being a type synonym. The presence of kind signature is not relevant to Trac #14888, so I'm just accepting the new output.